scripts/get_latest_build_sha.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+# standard bash error handling
+set -o nounset  # treat unset variables as an error and exit immediately.
+set -o errexit  # exit immediately when a command fails.
+set -E          # needs to be set if we want the ERR trap
+set -o pipefail # prevents errors in a pipeline from being masked
+
+last_build_id=$(curl -f -L -X GET https://storage.googleapis.com/kyma-prow-logs/logs/post-api-gateway-manager-build/latest-build.txt)
```

Comment:
I think istio should not use `post-api-gateway-manager-build`.
This approach might not be possible when moving to ADO-based pipeline. But I guess we will have to handle it then.